src/en/administration/from_scratch.md

```diff
@@ -0,0 +1,886 @@
+# From Scratch 
+
+> ⚠️ **Disclaimer:** this installation method is not recommended by the Lemmy developers. If you have any problems, you need to solve them yourself or ask the respective authors. If you notice any Lemmy bugs on an instance installed like this, please mention it in the bug report.
+
+## Installing Lemmy from source code
+
+Instructions for installing Lemmy natively, without relyng on Docker. Originally posted on [Some resources on setting Lemmy up from source - Lemmy dot C.A.](https://lemmy.ca/post/1066) **The current transcription has been adapted to improve readability**.
+
+### Important
+Note that building Lemmy requires a lot of hardware resources.  If you want to run Lemmy on a small VPS with very limited RAM (which seems like a perfectly acceptable way to run a production instance), maybe stick with the Docker image, or do your code building on another system that has more RAM.  RAM is huge with rust builds.
+
+Tag/release versions included in this note come from lemmy/docker/prod/docker-compose.yml and were current at the time this document was created.  **Definitely adjust to appropriate versions as necessary**.
+
+I had to switch from using `sudo` to `su`'ing in some places as the user was doing something odd/incomplete with the sudo env for pictrs
+
+## Setup
+
+| Dependencies                 |                                           |
+|------------------------------|-------------------------------------------|
+| app-admin                    | sudo                                      |
+| dev-vcs                      | [git](https://git-scm.com/)               |
+| dev-lang                     | [rust](https://www.rust-lang.org/)        |
+| dev-db                       | [postgresql](https://www.postgresql.org/) |
+| www-servers                  | [nginx](https://nginx.org/en/)            |
+| sys-apps                     | [yarn](https://yarnpkg.com/)              |
+| (optional) app-accessibility | espeak                                    |
+| app-shells                   | bash-completion                           |
+
+
+### Get postgresql service up & running
+
+#### Gentoo
```

Comment:
I would just mention Gentoo once at the top, then you dont have to repeat it every time. And other distros would be in completely separate sections.